static/sass/_snapcraft_l-application.scss

```diff
@@ -228,51 +229,8 @@
     padding-left: 4rem;
   }
 
-  .is-collapsed {
-    .hide-collapsed {
-      .p-side-navigation__icon {
-        display: none !important;
-      }
-
-      [aria-current="page"] {
-        &.p-side-navigation__link {
-          background-color: inherit !important;
-
-          &:hover {
-            background-color: inherit !important;
-            cursor: default !important;
-          }
-
-          &::before {
-            display: none !important;
-          }
-        }
-      }
-    }
-  }
-
-  @media (min-width: $breakpoint-small) and (max-width: $breakpoint-large) {
-    .l-navigation:not(.is-pinned, :hover) {
-      .hide-collapsed {
-        .p-side-navigation__icon {
-          display: none !important;
-        }
-
-        [aria-current="page"] {
-          &.p-side-navigation__link {
-            background-color: inherit !important;
-
-            &:hover {
-              background-color: inherit !important;
-              cursor: default !important;
-            }
-
-            &::before {
-              display: none !important;
-            }
-          }
-        }
-      }
-    }
+  .p-tooltip--right {
```

Comment:
Another very specific override. If we happen to change tooltip to any other variant it will stop working. 

If there are issues with z-index it would be good to identify them at source, or make this override better documented and applicable to any tooltip in the side nav.